dashboard/app/views/plc/learning_modules/new_learning_resource_for_module.html.haml

```diff
@@ -0,0 +1,37 @@
+- content_for(:head) do
+  = stylesheet_link_tag 'css/levelbuilder', media: 'all'
+
+%h3{style: 'margin-bottom: 24px;'}
+  Add a resource (Module: #{@learning_module.name})
+
+.new_resource_section{style: 'float: left; width: 33%'}
+  %p
+    = link_to 'New Script', new_script_path, class: 'btn'
```

Comment:
Okay - since two pages haven't been completed yet, I talked with Ryan and "New script" will link the user to create a new plc_task with type "ScriptCompletionTask" and "New Level" will link the user to create a new plc_task with type "LearningResourceTask." Eventually the creation pages for those tasks will appear different, but for now its enough to unblock Brook.